src/sonic-ctrmgrd/tests/kube_commands_test.py

```diff
@@ -0,0 +1,396 @@
+    },
+    4: {
+        common_test.DESCR: "Regular join: fail file update",
+        common_test.RETVAL: -1,
+        common_test.ARGS: ["10.3.157.24", 6443, False, False],
+        common_test.PROC_CMD: [
+            'sed *',
+            'rm -f *',
+            "kubectl --kubeconfig {} --request-timeout 20s drain None \
+--ignore-daemonsets".format(KUBE_ADMIN_CONF),
+            "kubectl --kubeconfig {} --request-timeout 20s delete node \
+None".format(KUBE_ADMIN_CONF),
+            "kubeadm reset -f",
+            "rm -rf /etc/cni/net.d",
+            "systemctl stop kubelet",
+            "modprobe br_netfilter",
+            "systemctl start kubelet",
+            "kubeadm join --discovery-file {} --node-name None".format(
+                KUBE_ADMIN_CONF)
+        ],
+        common_test.PROC_RUN: [True, True],
+        common_test.PROC_FAIL: [True],
+        common_test.PROC_THROW: [True]
+    },
+    5: {
+        common_test.DESCR: "Regular join: fail due to unable to lock",
+        common_test.RETVAL: -1,
+        common_test.ARGS: ["10.3.157.24", 6443, False, False],
+        common_test.FAIL_LOCK: True
+    }
+}
+
+
+reset_test_data = {
+    0: {
+        common_test.DESCR: "non force reset",
+        common_test.RETVAL: 0,
+        common_test.DO_JOIN: True,
+        common_test.ARGS: [False],
+        common_test.PROC_CMD: [
+            "kubectl --kubeconfig {} --request-timeout 20s drain None \
+--ignore-daemonsets".format(KUBE_ADMIN_CONF),
+            "kubectl --kubeconfig {} --request-timeout 20s delete node \
+None".format(KUBE_ADMIN_CONF),
+            "kubeadm reset -f",
+            "rm -rf /etc/cni/net.d",
+            "rm -f {}".format(KUBE_ADMIN_CONF),
+            "systemctl stop kubelet"
+        ]
+    },
+    1: {
+        common_test.DESCR: "force reset",
+        common_test.RETVAL: 0,
+        common_test.ARGS: [False],
+        common_test.PROC_CMD: [
+            "kubectl --kubeconfig {} --request-timeout 20s drain None \
+--ignore-daemonsets".format(KUBE_ADMIN_CONF),
+            "kubectl --kubeconfig {} --request-timeout 20s delete node \
+None".format(KUBE_ADMIN_CONF),
+            "kubeadm reset -f",
+            "rm -rf /etc/cni/net.d",
+            "rm -f {}".format(KUBE_ADMIN_CONF),
+            "systemctl stop kubelet"
+        ]
+    },
+    1: {
+        common_test.DESCR: "force reset",
+        common_test.RETVAL: 0,
+        common_test.ARGS: [True],
+        common_test.PROC_CMD: [
+            "kubeadm reset -f",
+            "rm -rf /etc/cni/net.d",
+            "rm -f {}".format(KUBE_ADMIN_CONF),
+            "systemctl stop kubelet"
+        ]
+    },
+    2: {
+        common_test.DESCR: "skip reset as not connected",
+        common_test.RETVAL: -1,
+        common_test.ARGS: [False],
+        common_test.PROC_CMD: [
+            "systemctl stop kubelet"
+        ]
+    }
+}
+
+class TestKubeCommands(object):
+
+    @classmethod
+    def setup_class(cls):
+        print("SETUP")
```

Comment:
`setup_class()` is a unittest concept. For pytest, fixtures should be used instead. Since this is basically a no-op, suggest removing.